examples/editor-assignment/src/utils/sidebar.ts

```diff
@@ -0,0 +1,27 @@
+import { PlainClientAPI } from 'contentful-management';
+import { AppExtensionSDK } from '@contentful/app-sdk';
+
+// Find all content types with the app assigned to the sidebar
+// and preselect them in the list of content type on initial load
+export const setInitialSidebarContentTypes = (
+  cma: PlainClientAPI,
+  sdk: AppExtensionSDK,
+  setSelectedCTs: (cts: string[]) => void
+) => {
+  (async () => {
```

Comment:
Why not make this an async function?

Reply:
It needed to be wrapped before but nice catch as we removed some returns but now this isn't needed